src/apps/concepts/api.ts

```diff
@@ -48,6 +49,7 @@ const api = {
             sortBy = "bestMatch",
             dataTypeFilters = [] as string[],
             classFilters = [] as string[],
+            generalFilters =[] as string[],
```

Comment:
```suggestion
            generalFilters = [],
```

The `as string[]` isn't necessary here.